<launch>

<group ns="px4_example">
	<node pkg="joy" name="joy_node" type="joy_node"/>
	<node pkg="px4" name="manual_input" type="manual_input"/>
	<node pkg="px4" name="attitude_estimator" type="attitude_estimator"/>
	<node pkg="px4" name="mc_att_control" type="mc_att_control"/>
</group>

</launch>
